docs/faq.rst

```diff
@@ -84,6 +84,27 @@ When viewing an individual tool or workflow, you can tell which versions are Ope
 
 :ref:`(back to top) <topFAQ>`
 
+.. _what-is-dockstore-generative-ai-policy:
+
+In human readable terms, what is Dockstore's approach to generative AI?
+-----------------------------------------------------------------------
+
+In broad strokes, we are exploring the use of generative AI to enhance the workflow and tool development process.
+For full transparency, we will work following these principles:
+
+1. For content generated and displayed to the public or to other users (e.g. AI generated topic sentences), we will only train or use as prompts, data that is already available publicly.
+In other words, as an example, only after the publish button is used, will we create topic sentences that are presented to other users.
```

Comment:
Regarding in-house AI usage, the deciding factor is whether the source data is public or private.  If we reword in those terms, we cover the scenario where we use AI on public data to generate information that we only display privately.  There's probably also a theoretical scenario where we could use AI on private data (to summarize it?) and generate something that was safe to display to the public...

Reply:
:thought_balloon: